sklearn/datasets/_covtype.py

```diff
@@ -80,7 +100,13 @@ def fetch_covtype(*, data_home=None, download_if_missing=True,
         If True, returns ``(data.data, data.target)`` instead of a Bunch
         object.
 
-        .. versionadded:: 0.20
```

Comment:
I don't think this should be removed.

Reply:
OK, I will add it back, thanks for pointing it out.